app/views/thredded/shared/_topic_nav.html.erb

```diff
@@ -2,19 +2,21 @@
   <%= yield :thredded_breadcrumbs %>
 
   <ul class="thredded--topic-navigation">
-    <% if messageboard.try(:persisted?) %>
+    <% if messageboard.try(:persisted?) -%>
```

Comment:
It's a new, unsaved record - like on messageboards#new

On Sunday, March 27, 2016, Shaun Butler <notifications@github.com> wrote:

> In app/views/thredded/shared/_topic_nav.html.erb
> <https://github.com/thredded/thredded/pull/166#discussion_r57527499>:
>
> > @@ -2,19 +2,21 @@
> >    <%= yield :thredded_breadcrumbs %>
> >
> >    <ul class="thredded--topic-navigation">
> > -    <% if messageboard.try(:persisted?) %>
> > +    <% if messageboard.try(:persisted?) -%>
>
> What's the scenario where you don't have a persisted messageboard?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/thredded/thredded/pull/166/files/1d76c31f52fd83f850436bb5231c1bcd44d71d6a#r57527499>
>